src/test/integration/fastapi/tests/test_fastapi.py

```diff
@@ -77,3 +94,80 @@ def test_requests_instrumentation(self):
         self.assertIsNotNone(children[0]["attributes"]["http.request.headers"])
         self.assertIsNotNone(children[0]["attributes"]["http.response.headers"])
         self.assertIsNotNone(children[0]["attributes"]["http.response.body"])
+
+    @skip_attr_size_limit
+    def test_large_span_attribute_size_default_max_size(self):
+        response = requests.get("http://localhost:8000/invoke-requests-large-response")
+        response.raise_for_status()
+
+        body = response.json()
+
+        assert body is not None
+
+        # TODO Do something deterministic
+        time.sleep(3)  # Sleep for two seconds to allow the exporter to catch up
+
+        spans_container = SpansContainer.get_spans_from_file()
+        self.assertEqual(4, len(spans_container.spans))
+
+        # assert root
+        root = spans_container.get_root()
+        self.assertIsNotNone(root)
+        root_attributes = root["attributes"]
+        self.assertEqual(root_attributes["http.status_code"], 200)
+        self.assertEqual(
+            root_attributes["http.url"],
+            "http://127.0.0.1:8000/invoke-requests-large-response",
+        )
+        self.assertEqual(len(root_attributes["http.method"]), 3)
```

Comment:
best to check the actual value, i.e., `"htt"`

Reply:
```
self.assertEqual(root_attributes["http.method"], "htt")
```

Reply:
I think it is `"GET"`.